src/egeoip.app.src

```diff
@@ -3,6 +3,12 @@
   {vsn,"1.1"},
   {modules,[]},
   {registered, []},
-  {env, [{dbfile, city}]},
+  {env, [
+      {dbfile, city}%,
+      % dbpath may be left undefined, so as to not run the self-updater
+      %   but if defined, it must be a dat.gz file
+      %{dbpath, "https://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz"}
+      % or e.g. "https://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
```

Comment:
There's no official statements of throttling; if it becomes an issue we can host it on our end anyway.